CHARTER.md

```diff
@@ -0,0 +1,238 @@
+Voting will by default close after 7 days. Any member of the TSC may request a 7 day extension for any reason, moving the closing date back by 7 days. Any member of the TSC may request additional extensions, approved at the discretion of any TSC chair.
+
+For a vote to carry, a quorum of 75% is required by default.
+
+If a TSC member wants to call for a vote that they wish to be non-public, they must do so by contacting the TSC Chairs directly.
+At the discretion of the TSC Chairs, a vote may be made non-public, and will then be handled by creating an Issue in the `TSC-private` repository.
+
+The topic and nature of non-public votes may remain non-public, including the results. (It is expected that vast majority of votes will be public. Non-public voting should only be used in exceptional circumstances.)
+
+#### Documenting decisions
+
+Either initially, or at any point during the process, any TSC member may suggest the issue being discussed is "significant or noteworthy." If there are no objections, the resolution actions for the issue must include the creation of an Any Decision Record (previously named Architectural Decision Record). The Any Decision Record (ADR) should include as much information as is thought to be useful, following the provided template. The Pull Request for the ADR must be approved by all those who were involved in the decision making process, which must also be documented in the ADR as the "deciders."
+
+(The quick consensus process does not require an Any Decision Record, but the decision should be minuted.)
+
+Non-public decisions should be documented (as an ADR or otherwise) in the private `TSC-private` repository.
+
+### 4.3: Other Project Roles
+
+The JSON Schema project recognizes the need for both technical and non-technical roles. While the OpenJS Foundation takes on business responsibilities as the legal entity hosting the project, there are other non-technical responsibilities.
+
+The TSC will look to create other roles as appropriate, and may update this document in accordance with the requirements for doing so, to formally recognize the additional roles.
+
+The following responsibilities are recognized as those requiring roles to be defined by the TSC:
+- Community and Industry connections
+- Brand awareness, recognition, and health
+- Strategic allocation of funds and budgeting for investment in JSON Schema and its ecosystem
+- Education and training
+- Financial allocation approval, tracking, and auditing
+
+## 5: Definitions
+
+The JSON Schema project: The project which is housed by the OpenJS Foundation which operates as The JSON Schema Org.
+
+The JSON Schema Org: The people, policies, processes, activities, and artifacts, found within the GitHub json-schema-org.
```

Comment:
The "project" encompasses all of the JSON Schema things, such as the Slack server and various social media profiles. The "Org" is what we use when referring to the GitHub organization specifically.

Maybe it isn't a useful distinction to have. I don't recall the specific event and reasoning that caused me to add it.

Reply:
The way I think about, the distinction is similar to a company and the products it produces. Using this analogy, the "JSON Schema Org" is the company and the "JSON Schema project" is the product the company produces. It's confusing because we have only one "product" and our "company" is named after the product, but I do see these as two separate concepts.

Reply:
So the fact that you have a different understanding of those terms and that that distinction isn't relevant within the charter, I'd just drop it off for now. From the perspective of the foundation (and thus this charter) the "JSON Schema Project" _is_ the organization in @jdesrosiers's example, so I'd just use that term or just "JSON Schema," but not "JSON Schema Org" which feels like the GitHub org (as @Relequestual alludes to above) and reads funny.

Reply:
> I'd just drop it off for now. From the perspective of the foundation (and thus this charter) the "JSON Schema Project" is the organization

I agree that the distinction probably isn't relevant and that it can be dropped, but I would expect it the other way around. I would expect that the "JSON Schema Org" is what is relevant to the foundation and the "JSON Schema project" is probably not relevant within the charter. For example, I would consider a Hyper-Schema spec, a JSON Schema form generation spec, a JSON Schema code generation spec, etc, to all be separate "products" from the JSON Schema validation/annotation spec we have now and I would expect all of those to fall under the foundation' umbrella should we choose to pursue them.

Of course, this is just arguing semantics of made up terms, so it doesn't really matter, but all that is just to say that I'd prefer "JSON Schema Org" over "JSON Schema project" as the one term that the charter uses.

Reply:
> Of course, this is just arguing semantics of made up terms, so it doesn't really matter, but all that is just to say that I'd prefer "JSON Schema Org" over "JSON Schema project" as the one term that the charter uses.

I totally understand why. However from the CPC’s charter and the bylaws, _project_ is the chosen term. So adopting a different term will lead to weirdness for policies and processes written at the CPC level (which, unsurprisingly, stands for Cross _Project_ Council).

Reply:
Understood. In that case, "project" is fine with me. Thanks for sharing your expertise.

Reply:
> Understood. In that case, "project" is fine with me. Thanks for sharing your expertise.

No, thanks for your understanding! It can be unpleasant to feel shoehorned into an ontology that doesn’t really match your collective representation of the space you’re operating in.

Reply:
> It's confusing because we have only one "product" and our "company" is named after the product, but I do see these as two separate concepts. - @jdesrosiers 

Like "Slack" the product vs "Slack" the company.

Reply:
Resolving this as I've removed the definitions.